### PR TITLE
Added installation code for non-emscripten bin, lib and includes

### PR DESCRIPTION
**Pull request by nyue**

### CODE REVIEW

**Comment by waldyrious** [comment]
I was about to suggest this! Just tested this PR branch locally and it works without issues.

File: CMakeLists.txt [hunk 1]
```diff
@@ -526,4 +526,31 @@ else ()
                         "${GTEST_SOURCE_DIR}/googletest/include")
     target_link_libraries(draco_tests draco gtest)
   endif ()
+
+  # Headers installation
+  file ( GLOB_RECURSE draco_headers RELATIVE ${CMAKE_SOURCE_DIR} "*.h" )
```
Comment (waldyrious) [inline CMakeLists.txt]: maybe `CMAKE_CURRENT_SOURCE_DIR` instead?

Comment (tomfinegan) [inline CMakeLists.txt]: We use ${draco_root} to refer to the root of the Draco tree in the Draco cmake files.

File: CMakeLists.txt [hunk 2]
```diff
@@ -526,4 +526,31 @@ else ()
...
+  foreach (filename ${draco_headers})
+    get_filename_component ( file_directory ${filename} DIRECTORY )
```
Comment (tomfinegan) [inline CMakeLists.txt]: style nit, please collapse unnecessary horizontal white space, this:
get_filename_component ( file_directory ${filename} DIRECTORY )

should be:
get_filename_component(file_directory ${filename} DIRECTORY)

File: CMakeLists.txt [hunk 3]
```diff
@@ -526,4 +526,31 @@ else ()
...
+    install ( FILES
```
Comment (tomfinegan) [inline CMakeLists.txt]: nit: This can fit on one, at max two lines.

Comment (tomfinegan) [inline CMakeLists.txt]: ditto on collapsing horizontal whitespace, same for the following commands. The foreach and endforeach are the only commands in here that should have a space between the command and the open parenthesis given the style throughout the rest of the cmake file.

File: CMakeLists.txt [hunk 4]
```diff
@@ -526,4 +526,31 @@ else ()
...
+      ${filename}
+      DESTINATION
+      ${CMAKE_INSTALL_PREFIX}/include/${file_directory}
```
Comment (tomfinegan) [inline CMakeLists.txt]: This should be include/draco/${file_directory}; I don't think we want the top level of the draco includes in /usr/local/include; we probably want them in /usr/local/include/draco.

Comment (tomfinegan) [inline CMakeLists.txt]: Closing PR due to no response on comments.

**Pull request closed without merging**